conf/app.routes

```diff
@@ -99,11 +99,15 @@ POST       /return-change-registration                          controllers.Retu
 
 GET        /remove-small-producer-confirm/:sdil                 controllers.RemoveSmallProducerConfirmController.onPageLoad(mode: Mode = NormalMode, sdil: String)
 POST       /remove-small-producer-confirm/:sdil                 controllers.RemoveSmallProducerConfirmController.onSubmit(mode: Mode = NormalMode, sdil: String)
+GET        /remove-small-producer-confirm/:sdil                 controllers.RemoveSmallProducerConfirmController.onPageLoad(mode: Mode = CheckMode, sdil: String)
```

Comment:
Are these duplicates of the one above - do we need a slightly different route name?